examples/model_compress/dependency_aware_pruner.py

```diff
@@ -0,0 +1,276 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
is it possible and suitable to merge this file into https://github.com/microsoft/nni/blob/master/examples/model_compress/model_prune_torch.py ?

Reply:
Sure